### PR TITLE
feat: spaces - resource list homepage

### PR DESCRIPTION
**Pull request by IrakliJani**
Closes: https://github.com/lightdash/lightdash/issues/3230

### Description:
Updated appearance of charts and dashboards on the homepage

localhost_3000_projects_3675b69e-8324-4110-bdca-059031aa8da3_home

### CODE REVIEW

**Comment by IrakliJani** [comment]
@PriPatel wdyt? I think card headings are too much for the homepage 🤔 

CleanShot 2022-09-14 at 11 37 53@2x

File: packages/frontend/package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         "@lightdash/common": "^0.251.2",
         "@sentry/react": "^6.17.6",
         "@sentry/tracing": "^6.17.6",
-        "@tanstack/react-table": "^8.5.11",
+        "@tanstack/react-table": "^8.5.13",
```
Comment (rephus) [inline packages/frontend/package.json:18]: what'st he reason for the upgrade ?

**Review by rephus — APPROVED** [review]
Code looks ok, I'm wondering if the id from here https://github.com/lightdash/lightdash/pull/3235/files#diff-caf21eb214b9b11df9b04f099619fdcb959111d855bcf024b63bf3ac7f3896c9R68 is going to cause issues if you mix both components on homepage...

**Comment by PriPatel** [comment]
@IrakliJani 

>  wdyt? I think card headings are too much for the homepage 🤔

do you mean theyre too big or need to not be in the cards itself?  Or there are too many? Thanks!

**Comment by PriPatel** [comment]
Also, I wonder - maybe you shouldn't need to sort the last updated dashboards and charts?

**Review by ZeRego — APPROVED** [review]

**Comment by IrakliJani** [comment]
@rephus 
> Code looks ok, I'm wondering if the id from here https://github.com/lightdash/lightdash/pull/3235/files#diff-caf21eb214b9b11df9b04f099619fdcb959111d855bcf024b63bf3ac7f3896c9R68 is going to cause issues if you mix both components on homepage...

id does not get applied to the dom element. it's just identifying a column (I use it to hide/show spaces column)

File: packages/frontend/package.json (hunk 1 above)
Comment (IrakliJani) [reply packages/frontend/package.json:18]: I'll revert this. I had a really weird bug and thought that would help.

**Comment by IrakliJani** [comment]
@PriPatel something was overriding. 16px looks great

**Pull request merged**